ts/screens/authentication/SpidInformationRequestScreen.tsx

```diff
@@ -121,4 +114,7 @@ const mapStateToProps = (state: GlobalState): ReduxMappedProps => ({
    */
   isFormValid: isValid(SPID_INFORMATION_FORM_NAME)(state)
 });
-export default connect(mapStateToProps)(SpidInformationRequestScreen);
+
+export default connect(mapStateToProps)(
+  withContextualHelp<Props>(SpidInformationRequestScreen)
```

Comment:
`withContextualHelp(SpidInformationRequestScreen)` should infer the needed types